services/crates/crates-downloads.service.js

```diff
@@ -0,0 +1,91 @@
+'use strict'
+
+const {
+  downloadCount: downloadCountColor,
+} = require('../../lib/color-formatters')
+const { metric } = require('../../lib/text-formatters')
+const { BaseCratesService, keywords } = require('./crates-base')
+
+module.exports = class CratesDownloads extends BaseCratesService {
+  static get category() {
+    return 'downloads'
+  }
+
+  static get route() {
+    return {
+      base: 'crates',
+      format: '(d|dv)/([A-Za-z0-9_-]+)(?:/([0-9.]+))?',
+      capture: ['which', 'crate', 'version'],
+    }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'Crates.io',
+        pattern: 'd/:crate',
+        exampleUrl: 'd/rustc-serialize',
```

Comment:
I'm in the process of converting all the uses of `exampleUrl`. Could you switch these to `namedParams`?